msprime/pedigrees.py

```diff
@@ -457,6 +458,31 @@ def sim_pedigree(
     end_time=None,
     direction="forward",
 ):
+    """
+    Simulates a pedigree
+
+    Arguments:
```

Comment:
Can you reformat this using sphinx :param: sections like the rest of the code please?